### PR TITLE
Fix #4 Addition of Issue template

### LINKED ISSUE #4: Issue Template
Add Issue Template

References:
https://github.com/oppia/oppia-android/tree/develop/.github/ISSUE_TEMPLATE
https://github.com/forem/forem/tree/main/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by vanamraghu**
Fixes #4
Issue Template consists of following files:

1. Bug Report
2. Feature Request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01-feature.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[Feature Request]: "
+labels: ["triage needed", "enhancement"]
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/01-feature.yml:4]: Remove labels from both the templates

File: .github/ISSUE_TEMPLATE/01-feature.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug in the Oppia project.
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/01-feature.yml:9]: Change project name.

File: .github/ISSUE_TEMPLATE/02-bug.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+name: Bug Report
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/02-bug.yml:1]: There is some error in this file fix that: https://github.com/rt4914/KlinicCon-Backend/blob/5ec1b1fb286869e7a5fa43e1c45a23190b36fcb8/.github/ISSUE_TEMPLATE/02-bug.yml

File: .github/ISSUE_TEMPLATE/02-bug.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/02-bug.yml:4]: Remove labels

File: .github/ISSUE_TEMPLATE/02-bug.yml (hunk 3 above)
Comment (rt4914) [reply .github/ISSUE_TEMPLATE/02-bug.yml:1]: Because of this error the UI is not getting generated correctly.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks for the PR.
Suggestions:
1. For future PRs, follow this for branch name: https://github.com/rt4914/KlinicCon-Backend/wiki#how-to-contribute
2. Fourth point in here: https://github.com/rt4914/KlinicCon-Backend/wiki#creating-a-pull-request
basically update the title and desciption to link it to issue number.

File: .github/ISSUE_TEMPLATE/01-feature.yml (hunk 1 above)
Comment (vanamraghu) [reply .github/ISSUE_TEMPLATE/01-feature.yml]: Removed it and pushed to latest PR

File: .github/ISSUE_TEMPLATE/01-feature.yml (hunk 2 above)
Comment (vanamraghu) [reply .github/ISSUE_TEMPLATE/01-feature.yml]: Fixed, changed to existing project name

File: .github/ISSUE_TEMPLATE/02-bug.yml (hunk 4 above)
Comment (vanamraghu) [reply .github/ISSUE_TEMPLATE/02-bug.yml]: Removed it

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM (stands for - Looks good to me.) Overall looks good, we can merge it now and later on update it as required.

Just one change update the title to:

`Fix #4 : Adds Issue template`

**Comment by vanamraghu** [comment]
> LGTM (stands for - Looks good to me.) Overall looks good, we can merge it now and later on update it as required.
> 
> Just one change update the title to:
> 
> `Fix #4 : Adds Issue template`

Updated the title

**Review by rt4914 — APPROVED** [review]
LGTM. Congratulations on merging your first PR 🚀 

Just make sure that you don't resolve comments in next PR. Person who has started the comments should resolve them.

**Pull request merged**